src/Analysis/Ast/Impl/Analyzer/Handlers/ImportHandler.cs

```diff
@@ -48,20 +52,21 @@ internal sealed partial class ImportHandler: StatementHandler {
                 }
 
                 var location = Eval.GetLoc(moduleImportExpression);
+                var moduleName = moduleImportExpression.MakeString();
                 IPythonModule module = null;
                 switch (imports) {
                     case ModuleImport moduleImport when moduleImport.FullName == Module.Name:
                         Eval.DeclareVariable(memberName, Module, VariableSource.Declaration, location);
                         break;
                     case ModuleImport moduleImport:
-                        module = await HandleImportAsync(node, moduleImport, cancellationToken);
+                        module = await HandleImportAsync(moduleImport, location, cancellationToken);
                         break;
                     case PossibleModuleImport possibleModuleImport:
-                        module = await HandlePossibleImportAsync(node, possibleModuleImport, cancellationToken);
+                        module = await HandlePossibleImportAsync(possibleModuleImport, moduleName, location, cancellationToken);
```

Comment:
We should use `PossibleModuleImport.PossibleModuleFullName` instead of `moduleName`. Do we use `DottedName.MakeString()` anywhere else?

Reply:
Not for analysis. It is used here and in `HoverSource` just to get the string to show. I update here as you suggested.